### ISSUE TITLE
Skipping network devices when using --run

### ISSUE THREAD

**Issue by Jean-Regis** [fuse-open/uno#206]
In a (relatively) recent change, uno is now skipping the adb network devices and this is a problem for us.

We use quite a lot of usb devices connected to the phones/tablets (for instance, customer displays), thus we need to push updates via wifi during development (the usb port has to be in host mode).

Would it be possible to have an option to toggle this feature ?

Thanks a lot :-)

**Comment by Jean-Regis** [comment]
Just noticed the commit:

https://github.com/fuse-open/uno/commit/44d54595cb229dab5a6f6443434a5873593ddadd#diff-bf9cb50a78e019e5cc72e8e42632c365#L46


Which says that the "installation will fail."

We made a bat script in the mean time that works fine over wifi, so I dont know why you wouldnt be able to replicate it.

```
uno build android
adb install -r <apk>
adb shell monkey -p "<package>" -c android.intent.category.LAUNCHER 1
adb logcat -c
adb logcat
```

**Comment by kusma** [comment]
Sounds like a good reason to revert 44d54595cb229dab5a6f6443434a5873593ddadd, I think...

**Comment by mortend** [comment]
Maybe we want a `--device <id>` option that can be given explicitly to install on a network device.

Or an `--all-devices` option to install the APK on all found devices, like it was before https://github.com/fuse-open/uno/commit/44d54595cb229dab5a6f6443434a5873593ddadd.

**Comment by kusma** [comment]
Sure, I just don't think a change in a network environment is a good justification to break other user's working setups. That's why I propose a revert.

**Comment by mortend** [comment]
We can revert and make a patch release to fix the regression, but that would reintroduce the problem I had because I don't control all devices on my network...

Maybe a better solution is to introduce `Android.IgnoreNetworkDevices` in `~/.unoconfig`. If we set it to `false` by default we'll fix the regression and I can fix my problem with a local config file.

**Comment by kusma** [comment]
@mortend: Yeah, making the new behavior opt-in instead seems like a better solution to me. I'm not saying we can't change the default either, but that should be a documented change in behavior in an upcoming feature-release if so, and not something we sneak into a patch-release.

**Comment by mortend** [comment]
@kusma: Do you think introducing `Android.IgnoreNetworkDevices` with default value set to `false` in a patch release is too sneaky?

I am also OK with reverting https://github.com/fuse-open/uno/commit/44d54595cb229dab5a6f6443434a5873593ddadd on `release-1.12` and introducing `Android.IgnoreNetworkDevices` on `master` instead.

**Comment by kusma** [comment]
@mortend: No, I consider that as restoring the old behavior, and I think that's probably the right trade-off.

But either way works for me, really :)

**Comment by mortend** [comment]
This was fixed in #207.
